public-docs/intro-glance.md

```diff
@@ -0,0 +1,80 @@
+3. [dom](https://github.com/reactioncommerce/reaction/blob/master/imports/plugins/core/dom/client/dom.js): Automatically adds `<meta>` tags are for products using the [dom](https://github.com/reactioncommerce/reaction/blob/release-1.8.0/imports/plugins/core/dom/client/dom.js) core package since 1.8, which uses the title, description and `details` fields of the product to render SEO-friendly data. You may need to use a tool like [SEO Inspector](https://chrome.google.com/webstore/detail/seo-inspector/iejckekdjogeeilmllnabmgkbbmedeal?hl=en) to see this data. Read more on [SEO and Metadata](seo-metadata.md).
+4. The storefront UI starter kit app uses NextJS framework with server side rendering enabled.
+
+Read more about [SEO and Reaction Commerce](https://blog.reactioncommerce.com/how-our-javascript-platform-handles-seo/) in our latest blog post.
+
+## Performance and Scaling
+
+We've done performance testing that mimics high-volume shops, and we're constantly working to improve performance and scalability both in terms of a production deployment and providing a fast and pleasant experience for developers.
+
+## Payments
+
+Reaction currently ships with one production-ready payment plugin: Stripe. If you use Stripe for payments, you can get up and running quickly. If you need to use a different payment service, you'll have to build a custom payment plugin to interface with their API, but we promise it is a painless procedure.
+
+## Interfaces / ETL
+
+The pluggable and schemaless nature of Reaction means that interfacing with your legacy or external systems involves little more than writing the transformations.
+
+## Customization
+
+- The Classic Meteor UI can be customized using theme plugins and provide Bootstrap UI themes.
+- The Starter Kit UI can be customized using a simple theme file, or you can easily swap entire React components for your own.
+- The Reaction API can be customized by plugins, including stitching your own GraphQL additions into the core schema or listening for core events and using them to trigger other actions.
+
+## Deployment
+
+You can deploy Reaction Commerce to anywhere you could deploy a Docker container or a Node app. Because of the complexities involved, we currently can’t offer any support for deployment.
+
+The best option depends on your budget and your expected level of traffic. Below are some options for deployment:
+
+- Galaxy: The Meteor Development Group offers a hosting service, Galaxy, that is focused on hosting Meteor apps with monitoring, etc. You can find more information [here](https://www.meteor.com/hosting).
```

Comment:
Galaxy is probably not a good hosting option anymore for this client/server version of the app

Reply:
I think it would work since Galaxy is just container hosting with some Meteor-specific sugar. But I don't necessarily think we need to recommend Galaxy because it's not going to be worth the cost for most people.